assets/js/components/GoogleChart.js

```diff
@@ -26,16 +26,21 @@ import '../util/initialize-google-global';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import { Chart } from 'react-google-charts';
+import { set } from 'lodash';
```

Comment:
Let's use the default import:
```suggestion
import set from 'lodash/set';
```